### DISCUSSION TITLE
迅雷网盘添加报错

### DISCUSSION THREAD

**Discussion by lazyyz** [AlistGo/alist#1096] [Invalid question]
alist 版本：2.5.2
迅雷网盘添加后，报错：ErrorCode: 0 ,Error: invalid_argument ,ErrorDescription: device_id required

**Comment by anwen-anyi** [comment]
现在把旧的迅雷删了,重现添加. 文档更新了你看下
文档链接：https://alist-doc.nn.ci/docs/driver/thunder

**Reply by lazyyz** [reply]
非常感谢，按照新文档，使用迅雷账号可以登录了。
